src/cascadia/TerminalSettingsEditor/MainPage.cpp

```diff
@@ -298,12 +298,11 @@ namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
         {
             auto profileVM{ _viewModelForProfile(_settingsClone.ProfileDefaults(), _settingsClone) };
             profileVM.IsBaseLayer(true);
-            _lastProfilesNavState = winrt::make<ProfilePageNavigationState>(profileVM,
-                                                                            _settingsClone.GlobalSettings().ColorSchemes(),
-                                                                            _lastProfilesNavState,
-                                                                            *this);
+            auto state{ winrt::make<ProfilePageNavigationState>(profileVM,

```

Comment:
You don't need to hold this as a member anymore?

Reply:
Nope! Now I moved the responsibility of tracking the last pivot value over to `ProfileViewModel` as a static property. Much cleaner!